scripts/examples/python/XenAPI/XenAPI.py

```diff
@@ -216,14 +216,15 @@ def _login(self, method, params):
             # pytype false positive: there is a socket.errno in both py2 and py3
             if e.errno == socket.errno.ETIMEDOUT: # pytype: disable=module-attr
                 raise xmlrpclib.Fault(504, 'The connection timed out')
-            else:
-                raise e
+            raise e
 
     def _logout(self):
         try:
             if self.last_login_method.startswith("slave_local"):
                 # Proxied function, pytype can't see it
-                return _parse_result(self.session.local_logout(self._session)) # pytype: disable=attribute-error
+                # pytype: disable=attribute-error
+                return _parse_result(self.session.local_logout(self._session))
+                # pytype: enable=attribute-error
```

Comment:
This line is repeated and looks like it's not needed

Reply:
One is enable, the other is disable. I cannot use just the disable because then pylint complains that the line is too long

Reply:
ahh, I see